crypto/keys/internal/ecdsa/pubkey_internal_test.go

```diff
@@ -0,0 +1,69 @@
+package ecdsa
+
+import (
+	"crypto/elliptic"
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+var secp256r1 = elliptic.P256()
```

Comment:
this test file seems duplicate, since everything tested here is also tested in `crypto/keys/secp256r1/pubkey_test.go`, is that correct?

Still ok to keep it though.

Reply:
Not much. they are testing on different levels (eg: secp marshaling goes through protobuf, but ecdsa marshaling tests only the bytes in/out without going through protobuf). The `secp256r1.PubKeyString` has some duplication though.